code/model/SubgoalPolicy.cpp

```diff
@@ -18,6 +18,7 @@ Subgoal::Subgoal (void)
 	p_PddlSubgoalPredicate = NULL;
 	b_IsLastSubgoalToTarget = false;
 	b_ForcedSequenceEnd = false;
+  b_isQuestion = false;
```

Comment:
it's weird. My new editor setup keeps messing up my indentation. Even though it looks all equally aligned in my editor..

W.e, i'll reindent through here